.github/workflows/ci.yml

```diff
@@ -48,7 +58,7 @@ jobs:
 
     - name: Run tests (macOS)
       if: runner.os == 'macOS'
-      run: opam exec -- node scripts/ciTest.js -all
+      run: opam exec -- node scripts/ciTest.js -ounit -mocha -theme
```

Comment:
Is this turning off build tests on x86 mac?

Reply:
++ @ostera

Reply:
keyed this by the `runner.name` now, so we keep running `-all` on `macos-latest`